acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +84,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
```

Comment:
woops. you are right.  sorry, i missed that it'll skip the assertions because of the switch. 

this is fine also